server/migrations/20170608140411-create-roles.js

```diff
@@ -0,0 +1,27 @@
+'use strict';
```

Comment:
There's another migrations directory in the project directory. What necessitates two migrations directories?